ui/lib/login/widget/login_form.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+class LoginForm extends StatefulWidget {
+  @override
+  LoginFormState createState() {
```

Comment:
nit: LoginFormState createState() => LoginFormState();

Reply:
I pulled this from the flutter docs, is this not best practice?